src/git-sync

```diff
@@ -26,7 +29,11 @@ function preconditions {
   if [ "$1" = "--all" ]; then
     sync_all=true
   elif [ "$(is_feature_branch "$INITIAL_BRANCH_NAME")" = false ]; then
+    # syncing a non-feature branch
```

Comment:
What do you think about switching the order and just commenting what condition the else is covering?
```bash
if [ "$1" = "--all" ]; then
  sync_all=true
elif [ "$(is_feature_branch "$INITIAL_BRANCH_NAME")" = true ]; then
  ensure_knows_parent_branches "$INITIAL_BRANCH_NAME"
else # non-feature branch
  should_push_tags=true
fi
```